### ISSUE TITLE
Add broker to allin1 container 

### ISSUE THREAD

**Issue by alberttwong** [StarRocks/starrocks#26060]
Usually it's on BE port 8000 but where is it running in the allin1? 

```
StarRocks > show broker
    -> ;
+-------------+-----------+------+-------+---------------+----------------+--------------------------------------------------------------------+
| Name        | IP        | Port | Alive | LastStartTime | LastUpdateTime | ErrMsg                                                             |
+-------------+-----------+------+-------+---------------+----------------+--------------------------------------------------------------------+
| local_load3 | 127.0.0.1 | 8060 | false | NULL          | NULL           |                                                                    |
| local_load2 | 127.0.0.1 | 8040 | false | NULL          | NULL           |                                                                    |
| local_load  | 127.0.0.1 | 8000 | false | NULL          | NULL           | java.net.ConnectException: Connection refused (Connection refused) |
| local_load  | 127.0.0.1 | 8040 | false | NULL          | NULL           |                                                                    |
| local_load  | 127.0.0.1 | 8060 | false | NULL          | NULL           |                                                                    |
+-------------+-----------+------+-------+---------------+----------------+--------------------------------------------------------------------+
5 rows in set (0.01 sec)
```

**Comment by alberttwong** [comment]
https://github.com/StarRocks/starrocks/issues/26061

**Comment by kevincai** [comment]
included since release 3.1.0
